include/stateMachineFwk/stateMachineEngine.h

```diff
@@ -31,7 +32,7 @@ namespace SM
 		static StateMachineEngine* Instance();
 	    void run();
       	bool addCBToProcQ(ControlBlock* cb);
-	
+	ActStatus handleProcedureEvent(ControlBlock* cb, State *currentState_p, Event& currentEvent);	
```

Comment:
2 arguments are taken as pointer and third as reference.. I thought we always prefer reference ? Can we start putting careful attention to all small/small details ?

Reply:
Changed to reference.